### PR TITLE
test: add basic test for s3 api

### PR DESCRIPTION
**Pull request by goshander**

### CODE REVIEW

File: test/api_s3.dart [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+    });
+
+    test('get buckets count', () async {
+      var bucketsCount = await api.getBucketsCount();
+
+      expect(bucketsCount, equals(1), reason: 'bucket exist');
+    });
+
+    test('upload - download - delete object', () async {
+      await api.deleteObject(objectPath);
```
Comment (AlexEbenrode) [inline test/api_s3.dart]: mb separate it to three independent tests?

**Review by AlexEbenrode — APPROVED** [review]

**Comment by AlexEbenrode** [comment]
image

What do u think about changing test filename for test locally?

**Comment by goshander** [comment]
> image
> 
> What do u think about changing test filename for test locally?

Oops, i didnt know about this magic)

File: test/api_s3.dart (hunk 1 above)
Comment (goshander) [reply test/api_s3.dart]: I thought about it, but before download something, we must upload something
And separated test after it useless

**Comment by goshander** [comment]
:tada: this issue has been resolved in version 1.2.0 :tada:

release is available on [GitHub](<github_release_url>)

**Pull request merged**